### PR TITLE
Fix tapScript doc rendering

### PR DESCRIPTION
**Pull request by SuperSandro2000**
see https://search.nixos.c3d2.de/?scope=6&option=microvm.host.tapScript

### CODE REVIEW

File: nixos-modules/host/options.nix [hunk 1]
```diff
@@ -37,9 +37,8 @@
         ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set "$id" up";
+      defaulText = "''${pkgs.iproute2}/bin/ip link set "$id" up";
```
Comment (SuperSandro2000) [inline nixos-modules/host/options.nix]:
```suggestion
      defaultText = "''${pkgs.iproute2}/bin/ip link set "$id" up";
```

**Review by astro — CHANGES_REQUESTED** [review]
Why did this PR have draft status?

File: nixos-modules/host/options.nix [hunk 2]
```diff
@@ -37,9 +37,8 @@
         ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set "$id" up";
+      defaultText = "''${pkgs.iproute2}/bin/ip link set "$id" up";
```
Comment (SuperSandro2000) [inline nixos-modules/host/options.nix]:
```suggestion
      defaultText = "''${pkgs.iproute2}/bin/ip link set \"$id\" up";
```

**Comment by SuperSandro2000** [comment]
> Why did this PR have draft status?

@codingmarcel is supposed to look, if it is now what we expect

File: nixos-modules/host/options.nix [hunk 3]
```diff
@@ -37,9 +37,8 @@
         ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set "$id" up";
```
Comment (MarcelCoding) [inline nixos-modules/host/options.nix]:
```suggestion
      default = "${pkgs.iproute2}/bin/ip link set \"$id\" up";
```

File: nixos-modules/host/options.nix [hunk 4]
```diff
@@ -37,9 +37,8 @@
         ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set \"$id\" up";
```
Comment (astro) [inline nixos-modules/host/options.nix]: Can we keep the `default = ''` with a newline, now that we have the `defaultText` too?

**Review by astro — CHANGES_REQUESTED** [review]

**Pull request merged**